server/controllers/userController.js

```diff
@@ -0,0 +1,39 @@
+import createToken from '../helpers/createToken'
+import bcrypt from 'bcrypt';
+import models from '../database/models';
+import responseMessage from '../helpers/responseMessage';
```

Comment:
this is not the right way to import from helpers. `import helpers from '../helpers'` then destructure what you want from helpers.

Reply:
done